src/main/java/nl/tudelft/contextproject/util/webinterface/EntityUtil.java

```diff
@@ -55,7 +57,10 @@ protected static JSONObject entityToJson(Entity entity) {
 		JSONObject json = new JSONObject();
 		json.put("x", Math.round(entity.getLocation().getX()));
 		json.put("y", Math.round(entity.getLocation().getZ()));
-		json.put("type", entity.getType().getWebId());
+		json.put("t", entity.getType().getWebId());
+		if (entity.getType() == EntityType.BOMB) {

```

Comment:
Nitpicking: It is faster to use `if (entity instanceof Bomb)` here. Since you are casting to Bomb below, this is also slightly safer.